src/Google.Api.Gax/Api/Gax/ClientHelper.cs

```diff
@@ -103,14 +103,13 @@ public static Channel CreateChannel(ServiceEndpointSettings endpoint, string def
         }
 
         /// <summary>
-        /// Creates a client using application default credentials, taking care of generic cloning and
-        /// credential scoping.
+        /// Creates a client using application default credentials, taking care of credential scoping.
```

Comment:
I would remove the parameter-specific documentation, and just repeat the "The behavior undefined if..."